packages/renderer/src/lib/kube/NamespaceDropdown.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+import { Dropdown } from '@podman-desktop/ui-svelte';
+
+import { kubernetesCurrentContextState } from '/@/stores/kubernetes-contexts-state';
+
+let namespacesPromise = $derived.by(async () => {
+  return $kubernetesCurrentContextState.reachable ? await window.kubernetesListNamespaces() : undefined;
+});
+
+let currentNamespacePromise = $derived.by(async () => {
+  return $kubernetesCurrentContextState.reachable ? await window.kubernetesGetCurrentNamespace() : '';
```

Comment:
```suggestion
  return $kubernetesCurrentContextState?.reachable ? await window.kubernetesGetCurrentNamespace() : '';
```

I suppose this will fix the problem?